src/interfaces/user.ts

```diff
@@ -0,0 +1,7 @@
+interface User {
+  name: string;
+  email: string;
+  password: string;
+}
+
+export default User;
```

Comment:
You can use the interface of User from `@prisma/client` too

Reply:
This One I have Made For an argument which is getting passed into a create user function. The problem with prisma client user is that it has id and , it will throw an error .
This Interface has been made just for one specific purpose.

Reply:
Okay fine